### ISSUE TITLE
Example user stories to guide our documentation

### ISSUE THREAD

**Issue by gabek** [gabek/fedidocs#13]
To guide what specifics we want to document and what problems we want to solve, let's share example user stories to cross reference as we're deciding what specific topics, protocols, recommendations and best practices to share.

**Comment by gabek** [comment]
From Johannes
> I want to follow Gabe on Mobilizon, so that I am notified in Mastodon when he next time calls for a meeting and I don't miss it.

**Comment by Johann150** [comment]
It might be interesting to also cover the basics. I think one such basic would be "I want to send a microblog message." which would be pretty common with existing fedi projects, perhaps it should even be specific to a receiving software because different softwares have different requirements.

**Comment by aschrijver** [comment]
I suggested this in some other places, but Use Cases can be specified in [Gherkin language](https://cucumber.io/docs/gherkin/reference/), including scenario's. Which can then be tested in apps, based on e.g. Cucumber or other BDD libs that exist for a particular language.

The Gherkin is a precise definition that could even be part of the Docs, while they form input for the Test Suite(s) at the same time.

Some example (though a very elaborate style of Gherkin scripting) is how DID ORB defines the [activitypub.feature](https://github.com/trustbloc/orb/blob/main/test/bdd/features/activitypub.feature).

**Comment by jernst** [comment]
Use cases are different from user stories.

**Comment by aschrijver** [comment]
Yes, I deliberately used the term. Gherkin use cases break down into user stories (well, depends on process you wanna follow too, of course)
